### ISSUE TITLE
Show folder name or alias under Current Repository drop-down

### ISSUE THREAD

**Issue by givihuda** [desktop/desktop#14125]
### Describe the feature or problem you’d like to solve
I'm working on an upgrade branch in a copy of the repo because it's been too difficult juggling vendor differences per the base language versions for the current version and the upgrade version. Vendor folders and files of course are not committed, but downloaded during package install, and sometimes I have to switch versions multiple times a day.

In GitHub Desktop (for Mac anyway), when I click the Current Repository drop-down, these two repos show the same name even though they're in different folders with different names. The only way I can distinguish them is to hover over their name to see the path. It's very easy to choose the wrong one or cumbersome to choose the right one.

### Proposed solution
Provide an option to set and show an alias, or show a distinguished folder or path name of the displayed repo name (maybe just an automatic "smart" path would be adequate).

Please share how this will it benefit Desktop and its users?

This is implicit and already stated in my previous paragraphs, but it will allow users to distinguish by glance which same-named repo they're selecting in different paths or folder names.

### Additional context

Please include any other context, like screenshots or mockups, if applicable.

I can't provide this without blurring out necessary information due to them being proprietary, but you should have no trouble visualizing my description or testing my scenario yourself.

**Comment by niik** [comment]
Hey @givihuda, thanks for reaching out!

> Provide an option to set and show an alias,

Great news! We do support setting custom aliases for repositories, you can access it by right-clicking on the repository and selecting 'Create Alias'

![image](https://user-images.githubusercontent.com/634063/157335261-7163fa19-4045-4b6d-9e3a-b79d6218b4e7.png)

I hope that addresses your issue, I'm going to close this out but feel free to comment if that doesn't solve your problem and we can take a closer look!

**Comment by givihuda** [comment]
OK thank you, that's helpful.

However, I don't consider that a fix, just a workaround. I may be asking for a new feature, I'm not sure.

Strangely, when I added another similar repo (like from initial remote `repo-name` but in a new git clone folder `repo-name-version-2`), GitHub Desktop did name it by the folder... but I think the reason may have been that I ran the git clone command naming the folder uniquely this time, whereas on the other repo I cloned it to a unique folder (different than the original repo clone) then renamed that new repo's folder.

So maybe GitHub Desktop recognized in the .git folder somehow the name of the CLI command folder name, and used that so I didn't have to set an alias in the app. Does this make sense to how GitHub Desktop works?

Anyway, I think a proper fix is simply to use the name of the folder as the name in GitHub Desktop, or if the folder is named differently than the remote repo, to prompt us to choose which one to use, or to use an alias... I don't know.

I don't think the workaround justifies closing the issue, because I think there's a proper fix that should be done that's more sensible and relies less on guessing or having counter-intuitive default behavior.
